src/pkgcheck/checks/git.py

```diff
@@ -318,19 +319,54 @@ def desc(self):
         return f'commit {self.commit}, missing sign-off{s}: {sign_offs}'
 
 
+class InvalidTagFormat(results.CommitResult, results.Error):
+    """Local commit has a tag that is incompliant.
+
+    Commit tags have restrictions as to the allowed format and data
+    used per GLEP 66 [#]_.
+
+    .. [#] https://www.gentoo.org/glep/glep-0076.html#certificate-of-origin
+    """
+
+    def __init__(self, tag, value, error, **kwargs):
+        super().__init__(**kwargs)
+        self.tag, self.value, self.error = tag, value, error
+
+    @property
+    def desc(self):
+        return f'commit {self.commit}, tag {self.tag} value {self.value}: {self.complaint}'
```

Comment:
Don't you mean to use self.error here instead of self.complaint?

Reply:
That I do.  I'm going to modify the test misc.assertReportSanity so it forces a str() of the report to catch things like this in addition to fixing what I screwed up here ;)